### PR TITLE
Feat: Added Scheduler and Jobs

### LINKED ISSUE #558: Create a view for Meetup updates (Backend)
## Description
As a meetup owner, I should be able to post updates to meetups.

As  a meetup attendee, for any RSVP'd meetings:
- Receive updates on Location Change
- Receive updates on Date Change
- Receive updates on Time Change
- Reminders for the event (1 day prior, 1 hour prior)

## Definition of Done
- [x] All of the required items are completed.
- [x] Approval by 1 mentor.

## Estimation
48 hours

### PR DESCRIPTION
**Pull request by ritwickraj78**
### Description
- Added Django-Apscheduler for Async Jobs
- Added Jobs to be executed

Fixes #558 

### Type of Change:
**Delete irrelevant options.**

- Code

**Code/Quality Assurance Only**
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?

Jobs are created successfully and are executed as per the Schedule.

### Checklist:
**Delete irrelevant options.**

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings 
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been published in downstream modules

### CODE REVIEW

File: systers_portal/meetup/signals.py [hunk 1]
```diff
@@ -37,3 +42,21 @@ def create_notice_types(sender, **kwargs):
                       ("your support request has been approved"))
     NoticeType.create("new_meetup_request", ("New Meetup Request"),
                       ("a user has added a meetup request"))
+
+
+@receiver(pre_save, sender=Meetup, dispatch_uid="location_change")
+def change_notify(sender, instance, **kwargs):
```
Comment (SanketDG) [inline systers_portal/meetup/signals.py]:
```suggestion
def notify_change(sender, instance, **kwargs):
```

File: systers_portal/users/scheduler.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        logging.basicConfig()
+        logging.getLogger('apscheduler').setLevel(logging.DEBUG)
+    register_events(scheduler)
+    scheduler.start()
+    meetup_list = Meetup.objects.filter(date__gte=datetime.date.today())
+    for meetup in meetup_list:
+        name = "Reminder for {0}".format(meetup.title)
+        scheduler.add_job(send_reminder, "date", run_date=meetup.date - timedelta(hours=1),
+                          args=[meetup], id=name, replace_existing=True)
+    print("scheduler started")
```
Comment (SanketDG) [inline systers_portal/users/scheduler.py]: can be removed

File: systers_portal/users/scheduler.py [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+        for member in community.members:
+            html_text = \
+                render_to_string("templates/community/weekly_digest_email.html",
+                                 {'user': member,
+                                  'count': count,
+                                  'community': community})
+            send_mail(
+                subject,
+                'Weekly Digest',
+                'from@example.com',
```
Comment (SanketDG) [inline systers_portal/users/scheduler.py]: let's use a single value from settings for all of these :)

File: systers_portal/users/scheduler.py [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+from django_apscheduler.jobstores import register_events, register_job
+
+from django.conf import settings
+
+from meetup.utils import send_reminder
+from meetup.models import Meetup
+
+from community.models import Community
+
+scheduler = BackgroundScheduler(settings.SCHEDULER_CONFIG)
```
Comment (SanketDG) [inline systers_portal/users/scheduler.py:19]: Will you define this?

**Review by SanketDG — CHANGES_REQUESTED** [review]
I thought this was somehow large, but it was the earlier commits making it so.

Nothing really too complex here, looks good except for a few minor ones.

File: systers_portal/users/scheduler.py (hunk 3 above)
Comment (ritwickraj78) [reply systers_portal/users/scheduler.py]: the from email?

File: systers_portal/users/scheduler.py (hunk 4 above)
Comment (ritwickraj78) [reply systers_portal/users/scheduler.py:19]: Already defined in base.py settings

File: systers_portal/users/scheduler.py (hunk 3 above)
Comment (SanketDG) [reply systers_portal/users/scheduler.py]: yes

File: systers_portal/meetup/signals.py (hunk 1 above)
Comment (ritwickraj78) [reply systers_portal/meetup/signals.py]: Done

File: systers_portal/users/scheduler.py (hunk 2 above)
Comment (ritwickraj78) [reply systers_portal/users/scheduler.py]: Done

File: systers_portal/users/scheduler.py (hunk 3 above)
Comment (ritwickraj78) [reply systers_portal/users/scheduler.py]: Done

**Review by SanketDG — COMMENTED** [review]
@ritwickraj78 I can not test this locally right now, did you test this throughly by actually sending the emails?

If not, @sakshi1499 @sammy1997 can you have a go at it? You can use any popular smtp server out there, GMail's the easiest.

Looks good to me, but I would also like to have some tests later (icebox is fine)

**Comment by ritwickraj78** [comment]
@SanketDG I tested on the console. Worked fine

**Review by sakshi1499 — DISMISSED** [review]
Honestly, Looks good to me. I haven't tested it locally so not sure about that. Rest looks fine :)

**Comment by SanketDG** [comment]
@ritwickraj78 Could you rebase this? Let's merge this today! :tada:

**Comment by ritwickraj78** [comment]
@SanketDG Already rebased

**Comment by SanketDG** [comment]
@ritwickraj78 There are conflicts I can see, a commit was merged recently.

**Comment by ritwickraj78** [comment]
@SanketDG Rebased!

**Review by SanketDG — APPROVED** [review]
LGTM!

**Review by sammy1997 — APPROVED** [review]
LGTM as well

**Pull request merged**